packages/create-vite/index.js

```diff
@@ -237,6 +263,46 @@ async function init() {
   // determine template
   template = variant || framework || template
 
+  const pkgInfo = pkgFromUserAgent(process.env.npm_config_user_agent)
+  const pkgManager = pkgInfo ? pkgInfo.name : 'npm'
+  const isYarn1 = pkgManager === 'yarn' && pkgInfo?.version.startsWith('1.')
+
+  if (template.startsWith('custom-')) {
+    const getCustomCommand = (name) => {
+      for (const f of FRAMEWORKS) {
+        for (const v of f.variants || []) {
+          if (v.name === name) {
+            return v.customCommand
+          }
+        }
+      }
+    }
+    const customCommand = getCustomCommand(template)
+    const fullCustomCommand = customCommand
+      .replace('TARGET_DIR', targetDir)
+      .replace(/^npm create/, `${pkgManager} create`)
+      // Only Yarn 1.x doesn't support `@version` in the `create` command
+      .replace('@latest', () => (isYarn1 ? '' : '@latest'))
+      .replace('^npm exec', () => {
```

Comment:
```suggestion
      .replace(/^npm exec/, () => {
```
This should be like this?